### PR TITLE
LPD-79679 Add an ObjectDefinition-agnostic getValuesListCount overload

### PR DESCRIPTION
**Pull request by Mikellorza**
## Summary

Add a new `ObjectEntryLocalService.getValuesListCount` overload that counts `ObjectEntry` rows across multiple `ObjectDefinition`s **without joining the per-definition dynamic tables**. The new path queries `ObjectEntryTable` directly with a caller-supplied `Predicate`.

```java
int getValuesListCount(
    Long[] groupIds, long companyId,
    Long[] objectDefinitionIds, Predicate predicate)
```

## Why

The existing overload (`getValuesListCount(... long objectDefinitionId, Predicate, boolean preferApproved, String search)`) is **per-definition**: it builds a `FROM` over the dynamic object-definition table, the extension table, and joins the localization table. That contract is right when filtering or sorting by a definition-specific custom field, but it has two costs that don't fit the use case we're working on:

1. **Many definitions, one count.** For LPD-79669 (CMS quick-filter chips: *In Draft*, *Expiring Soon*, *Expired*, *Review Date Overdue*) we need a single number that aggregates entries across all CMS `ObjectDefinition`s in a depot. Calling the existing overload N times and summing would be N queries with N×(dynamic+extension+localization) joins per chip — not viable on a depot with several CMS definitions.

2. **The predicate doesn't need definition columns.** All the CMS quick-filter predicates only reference columns that are native to `ObjectEntryTable` itself (`status`, `expirationDate`, `reviewDate`). The dynamic/extension/localization joins are dead weight for this use case.

This overload addresses both: a single count, across an arbitrary set of definitions, against `ObjectEntryTable` columns only.

## How

- New public method on `ObjectEntryLocalServiceImpl` (regenerated into the `Util` / `Wrapper` / interface via Service Builder).
- New private helper `_getObjectEntriesGroupByStepWithoutDefinition` whose `FROM` is `ObjectEntryTable.INSTANCE` (no dynamic-table joins). Where clause keeps the standard invariants: `companyId`, `groupId IN (...)`, `objectDefinitionId IN (...)`, `status != IN_TRASH`, root-entry restriction, `_getHeadObjectEntryPredicate(false)`, and the user-supplied `predicate`.
- Permission filtering via new `_getPermissionWherePredicateForDefinitions(groupIds, objectDefinitionIds)`: for each `objectDefinitionId`, an `(objectDefinitionId = X) AND <permX>` branch is OR-ed. The per-definition `_getPermissionWherePredicate(...)` is reused.

## Limitation

This overload does **not** support `ObjectDefinition`s with `accountEntryRestricted = true`. The existing `_getPermissionWherePredicate` builds a predicate that references a column of the per-definition dynamic table (the account-entry-restricted custom field). That column is not in the `FROM` clause of the new query, so the resulting SQL would be invalid at runtime. The CMS object definitions we care about today are not account-restricted; if that changes the path will have to be revised (skip that branch, or add per-definition joins).

## Test plan

New integration test `ObjectEntryLocalServiceTest.testGetValuesListCountByObjectDefinitionIds`:

- Creates three site-scoped `ObjectDefinition`s (two relevant + one used to verify isolation) plus an "irrelevant" `Group`.
- Seeds nine entries covering: `expiringSoon` (APPROVED, expDate = now+3d), an entry beyond the 7-day window, an already-expired-by-date entry, `DRAFT`, `EXPIRED`, `reviewDate` overdue / future, and exclusion cases (entry on the irrelevant definition, entry on the irrelevant group).
- Asserts the new method returns the expected count for each of the four CMS quick-filter predicates: `status = DRAFT`, `status = EXPIRED`, `status = APPROVED AND expirationDate > now AND expirationDate <= now+7d`, and `reviewDate < now`.
- Asserts isolation by `objectDefinitionIds` (irrelevant definition only counts when explicitly passed) and by `groupIds` (irrelevant group only counts when explicitly passed).

`gw testIntegration --tests com.liferay.object.service.test.ObjectEntryLocalServiceTest.testGetValuesListCountByObjectDefinitionIds` passes locally.

## Commits

1. `LPD-79679 Add an ObjectDefinition-agnostic getValuesListCount overload` — public method + private helpers in `ObjectEntryLocalServiceImpl`.
2. `LPD-79679 buildService` — Service Builder regeneration of `ObjectEntryLocalService`, `ObjectEntryLocalServiceUtil`, `ObjectEntryLocalServiceWrapper`.
3. `LPD-79679 Add test` — the integration test described above.
4. `LPD-79679 Semver` — `Bundle-Version` 123.0.0 → 123.1.0 and `packageinfo` 85.0.0 → 85.1.0 (minor: new public API surface).

### CODE REVIEW

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -1867,6 +1867,19 @@ public int getValuesListCount(
 				predicate, preferApproved, search));
 	}
 
+	public int getValuesListCount(
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:1868]: You missed the `@override` notation.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 2]
```diff
@@ -4723,6 +4736,53 @@ private GroupByStep _getObjectEntriesGroupByStep(
 		);
 	}
 
+	private GroupByStep _getObjectEntriesGroupByStepWithoutDefinition(
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: Since this method has different parameters from the existing `_getObjectEntriesGroupByStep`, Java's method overloading lets us reuse the same name. Could you rename `_getObjectEntriesGroupByStepWithoutDefinition` to just `_getObjectEntriesGroupByStep`?

Simpler is always the better choice in Liferay 😄

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 3]
```diff
@@ -1867,6 +1867,19 @@ public int getValuesListCount(
 				predicate, preferApproved, search));
 	}
 
+	public int getValuesListCount(
+			Long[] groupIds, long companyId, Long[] objectDefinitionIds,
+			Predicate predicate)
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: I know the existing overloads do not follow alphabetical ordering, but I would sort the parameters of this new method alphabetically while we are introducing it.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 4]
```diff
@@ -4723,6 +4736,53 @@ private GroupByStep _getObjectEntriesGroupByStep(
 		);
 	}
 
+	private GroupByStep _getObjectEntriesGroupByStepWithoutDefinition(
+			Long[] groupIds, long companyId, FromStep fromStep,
+			Long[] objectDefinitionIds, Predicate predicate)
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: I know the existing overloads do not follow alphabetical ordering, but I would sort the parameters of this new method alphabetically while we are introducing it.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 5]
```diff
@@ -4935,6 +4995,52 @@ private Predicate _getPermissionWherePredicate(
 		return permissionWherePredicate;
 	}
 
+	private Predicate _getPermissionWherePredicateForDefinitions(
+			Long[] groupIds, Long[] objectDefinitionIds)
+		throws PortalException {
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: Since this method has different parameters from the existing `_getPermissionWherePredicate`, Java's method overloading lets us reuse the same name. Could you rename `_getPermissionWherePredicateForDefinitions` to just `_getPermissionWherePredicate`?

Simpler is always the better choice in Liferay 😄

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 6]
```diff
@@ -4935,6 +4995,52 @@ private Predicate _getPermissionWherePredicate(
...
+
+			return null;
+		}
+
+		Predicate permissionWherePredicate = null;
+
+		for (Long objectDefinitionId : objectDefinitionIds) {
+			ObjectDefinition objectDefinition =
+				_objectDefinitionPersistence.findByPrimaryKey(
+					objectDefinitionId);
```
[the remark this answers is not in the document]
Comment (marcelabc) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:5014]: I noticed this too. Since this loop already iterates over each `ObjectDefinition`, it is the natural place to add the `accountEntryRestricted` validation (no extra query needed). What do you think?

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 7]
```diff
@@ -4935,6 +4995,52 @@ private Predicate _getPermissionWherePredicate(
...
+
+			Predicate definitionPermissionPredicate =
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: Could you rename `definitionPermissionPredicate` to `objectDefinitionPermissionWherePredicate`? 
Two points: use the full entity name (`ObjectDefinition`, not `Definition`), and align the variable name with the method it comes from (`_getPermissionWherePredicate` returns a `permissionWherePredicate`).

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 8]
```diff
@@ -4935,6 +4995,52 @@ private Predicate _getPermissionWherePredicate(
...
+				_getPermissionWherePredicate(
+					DynamicObjectDefinitionTableUtil.
+						getDynamicObjectDefinitionTable(
+							false, objectDefinition, _objectFieldLocalService),
+					groupIds);
+
+			Predicate scopedPredicate =
+				ObjectEntryTable.INSTANCE.objectDefinitionId.eq(
+					objectDefinitionId);
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: Could you rename `scopedPredicate` to  `objectDefinitionPredicate`? 
Two reasons:

1. In Liferay, "scope" refers to the company/site/depot scope of an object definition, not to filtering by an ID — so `scoped` is misleading here.
2. `objectDefinitionPredicate` reflects the variable's actual role: the OR branch corresponding to one `ObjectDefinition` (ID match plus the per-definition permission predicate).

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 9]
```diff
@@ -4935,6 +4995,52 @@ private Predicate _getPermissionWherePredicate(
 		return permissionWherePredicate;
 	}
 
+	private Predicate _getPermissionWherePredicateForDefinitions(
+			Long[] groupIds, Long[] objectDefinitionIds)
+		throws PortalException {
+
+		if ((PermissionThreadLocal.getPermissionChecker() == null) ||
+			ArrayUtil.isEmpty(groupIds) ||
+			ArrayUtil.isEmpty(objectDefinitionIds)) {
```
Comment (marcelabc) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: Could you sort the conditions alphabetically?

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 10]
```diff
@@ -5577,6 +5578,223 @@ public void testGetValuesList() throws Exception {
...
+				Collections.singletonList(
+					ObjectFieldUtil.createObjectField(
+						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+						ObjectFieldConstants.DB_TYPE_STRING,
+						RandomTestUtil.randomString(), "name")),
+				ObjectDefinitionConstants.SCOPE_SITE);
+
+		Group irrelevantGroup = null;
+
+		try {
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java]: Could you remove the `try/finally` block? I noticed that Brian does not approve this pattern (only in edge cases). You can move the cleanup right after the assertions instead.

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 11]
```diff
@@ -5577,6 +5578,223 @@ public void testGetValuesList() throws Exception {
 			24, values3, valuesList.get(2), selectedObjectFieldNames);
 	}
 
+	@Test
+	public void testGetValuesListCountByObjectDefinitionIds() throws Exception {
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:5582]: I was thinking that this method could be refactor to something similar to this:

```
	@Test
	public void testGetValuesListCountByObjectDefinitionIds() throws Exception {

		// Add object entry on irrelevant group

		Group irrelevantGroup = GroupTestUtil.addGroup();

		ObjectDefinition objectDefinition1 =
			ObjectDefinitionTestUtil.publishObjectDefinition(
				Collections.singletonList(
					ObjectFieldUtil.createObjectField(
						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
						ObjectFieldConstants.DB_TYPE_STRING,
						RandomTestUtil.randomString(), "name")),
				ObjectDefinitionConstants.SCOPE_SITE);

		ObjectEntry objectEntry1 = _addObjectEntry(
			irrelevantGroup.getGroupId(),
			objectDefinition1.getObjectDefinitionId(),
			HashMapBuilder.<String, Serializable>put(
				"name", RandomTestUtil.randomString()
			).build());

		ServiceContext serviceContext =
			ServiceContextTestUtil.getServiceContext();

		_objectEntryLocalService.updateStatus(
			TestPropsValues.getUserId(), objectEntry1.getObjectEntryId(),
			WorkflowConstants.STATUS_DRAFT, serviceContext);

		// Add object entry on irrelevant object definition

		ObjectDefinition irrelevantObjectDefinition =
			ObjectDefinitionTestUtil.publishObjectDefinition(
				Collections.singletonList(
					ObjectFieldUtil.createObjectField(
						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
						ObjectFieldConstants.DB_TYPE_STRING,
						RandomTestUtil.randomString(), "name")),
				ObjectDefinitionConstants.SCOPE_SITE);

		ObjectEntry objectEntry2 = _addObjectEntry(
			TestPropsValues.getGroupId(),
			irrelevantObjectDefinition.getObjectDefinitionId(),
			HashMapBuilder.<String, Serializable>put(
				"name", RandomTestUtil.randomString()
			).build());

		_objectEntryLocalService.updateStatus(
			TestPropsValues.getUserId(), objectEntry2.getObjectEntryId(),
			WorkflowConstants.STATUS_DRAFT, serviceContext);

		// Add object entry with distant expiration date

		ObjectEntry objectEntry3 = _addObjectEntry(
			TestPropsValues.getGroupId(),
			objectDefinition1.getObjectDefinitionId(),
			HashMapBuilder.<String, Serializable>put(
				"name", RandomTestUtil.randomString()
			).build());

		Date now = new Date();

		objectEntry3.setExpirationDate(
			new Date(now.getTime() + (10 * Time.DAY)));

		_objectEntryLocalService.updateObjectEntry(objectEntry3);

		// Add object entry with future review date

		ObjectDefinition objectDefinition2 =
			ObjectDefinitionTestUtil.publishObjectDefinition(
				Collections.singletonList(
					ObjectFieldUtil.createObjectField(
						ObjectFieldConstants.BUSINESS_TYPE_TEXT,
						ObjectFieldConstants.DB_TYPE_STRING,
						RandomTestUtil.randomString(), "name")),
				ObjectDefinitionConstants.SCOPE_SITE);

		ObjectEntry objectEntry4 = _addObjectEntry(
			TestPropsValues.getGroupId(),
			objectDefinition2.getObjectDefinitionId(),
			HashMapBuilder.<String, Serializable>put(
				"name", RandomTestUtil.randomString()
			).build());

		objectEntry4.setReviewDate(new Date(now.getTime() + (5 * Time.DAY)));

		_objectEntryLocalService.updateObjectEntry(objectEntry4);

		// Add object entry with imminent expiration date

		ObjectEntry objectEntry5 = _addObjectEntry(
			TestPropsValues.getGroupId(),
			objectDefinition1.getObjectDefinitionId(),
			HashMapBuilder.<String, Serializable>put(
				"name", RandomTestUtil.randomString()
			).build());

		objectEntry5.setExpirationDate(
			new Date(now.getTime() + (3 * Time.DAY)));

		_objectEntryLocalService.updateObjectEntry(objectEntry5);

		// Add object entry with overdue expiration date

		ObjectEntry objectEntry6 = _addObjectEntry(
			TestPropsValues.getGroupId(),
			objectDefinition1.getObjectDefinitionId(),
			HashMapBuilder.<String, Serializable>put(
				"name", RandomTestUtil.randomString()
			).build());

		objectEntry6.setExpirationDate(new Date(now.getTime() - Time.DAY));

		_objectEntryLocalService.updateObjectEntry(objectEntry6);

		// Add object entry with overdue review date

		ObjectEntry objectEntry7 = _addObjectEntry(
			TestPropsValues.getGroupId(),
			objectDefinition2.getObjectDefinitionId(),
			HashMapBuilder.<String, Serializable>put(
				"name", RandomTestUtil.randomString()
			).build());

		objectEntry7.setReviewDate(new Date(now.getTime() - (2 * Time.DAY)));

		_objectEntryLocalService.updateObjectEntry(objectEntry7);

		// Add object entry with status draft

		ObjectEntry objectEntry8 = _addObjectEntry(
			TestPropsValues.getGroupId(),
			objectDefinition2.getObjectDefinitionId(),
			HashMapBuilder.<String, Serializable>put(
				"name", RandomTestUtil.randomString()
			).build());

		_objectEntryLocalService.updateStatus(
			TestPropsValues.getUserId(), objectEntry8.getObjectEntryId(),
			WorkflowConstants.STATUS_DRAFT, serviceContext);

		// Add object entry with status expired

		ObjectEntry objectEntry9 = _addObjectEntry(
			TestPropsValues.getGroupId(),
			objectDefinition2.getObjectDefinitionId(),
			HashMapBuilder.<String, Serializable>put(
				"name", RandomTestUtil.randomString()
			).build());

		_objectEntryLocalService.updateStatus(
			TestPropsValues.getUserId(), objectEntry9.getObjectEntryId(),
			WorkflowConstants.STATUS_EXPIRED, serviceContext);

		Long[] groupIds = {TestPropsValues.getGroupId()};
		Long[] objectDefinitionIds = {
			objectDefinition1.getObjectDefinitionId(),
			objectDefinition2.getObjectDefinitionId()
		};

		// Predicate with imminent expiration date

		Assert.assertEquals(
			1,
			_objectEntryLocalService.getValuesListCount(
				groupIds, TestPropsValues.getCompanyId(), objectDefinitionIds,
				ObjectEntryTable.INSTANCE.status.eq(
					WorkflowConstants.STATUS_APPROVED
				).and(
					ObjectEntryTable.INSTANCE.expirationDate.gt(now)
				).and(
					ObjectEntryTable.INSTANCE.expirationDate.lte(
						new Date(now.getTime() + (7 * Time.DAY)))
				)));

		// Predicate with irrelevant group

		Assert.assertEquals(
			1,
			_objectEntryLocalService.getValuesListCount(
				new Long[] {irrelevantGroup.getGroupId()},
				TestPropsValues.getCompanyId(), objectDefinitionIds,
				ObjectEntryTable.INSTANCE.status.eq(
					WorkflowConstants.STATUS_DRAFT)));

		// Predicate with irrelevant object definition

		Assert.assertEquals(
			1,
			_objectEntryLocalService.getValuesListCount(
				groupIds, TestPropsValues.getCompanyId(),
				new Long[] {irrelevantObjectDefinition.getObjectDefinitionId()},
				ObjectEntryTable.INSTANCE.status.eq(
					WorkflowConstants.STATUS_DRAFT)));

		// Predicate with overdue review date

		Assert.assertEquals(
			1,
			_objectEntryLocalService.getValuesListCount(
				groupIds, TestPropsValues.getCompanyId(), objectDefinitionIds,
				ObjectEntryTable.INSTANCE.reviewDate.lt(now)));

		// Predicate with status draft

		Assert.assertEquals(
			1,
			_objectEntryLocalService.getValuesListCount(
				groupIds, TestPropsValues.getCompanyId(), objectDefinitionIds,
				ObjectEntryTable.INSTANCE.status.eq(
					WorkflowConstants.STATUS_DRAFT)));

		// Predicate with status expired

		Assert.assertEquals(
			1,
			_objectEntryLocalService.getValuesListCount(
				groupIds, TestPropsValues.getCompanyId(), objectDefinitionIds,
				ObjectEntryTable.INSTANCE.status.eq(
					WorkflowConstants.STATUS_EXPIRED)));

		
		_objectDefinitionLocalService.deleteObjectDefinition(objectDefinition1);
		_objectDefinitionLocalService.deleteObjectDefinition(objectDefinition2);
		_objectDefinitionLocalService.deleteObjectDefinition(
			irrelevantObjectDefinition);

		GroupTestUtil.deleteGroup(irrelevantGroup);
	}
```

Can you analyze and see if you would change/improve something?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 6 above)
[the remark this answers is not in the document]
Comment (marcelabc) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:5014]: cc: @Mikellorza

**Comment by marcelabc** [comment]
Hey @Mikellorza, could you run the `validateBranch` command and let me know if it passes?

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 1 above)
Comment (Mikellorza) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:1868]: thanks :)

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 2 above)
Comment (Mikellorza) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: yes of corse ,thanks  😄

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 5 above)
Comment (Mikellorza) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: ok , no problem, thanks @marcelabc

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 3 above)
Comment (Mikellorza) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: thanks

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 9 above)
Comment (Mikellorza) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: yes, thanks

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 6 above)
[the remark this answers is not in the document]
Comment (Mikellorza) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:5014]: great, thanks @marcelabc

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 7 above)
Comment (Mikellorza) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: done!

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 8 above)
Comment (Mikellorza) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: done

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java (hunk 11 above)
Comment (Mikellorza) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:5582]: Yes, it looks good and it becomes more descriptive. 

thanks a lot @marcelabc

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java (hunk 10 above)
Comment (Mikellorza) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java]: done! :)

**Comment by Mikellorza** [comment]
> Hey @Mikellorza, could you run the `validateBranch` command and let me know if it passes?

Hi @marcelabc ,

I have applied all your suggestions and the validateBranch command passes.

thanks for your work

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
@Mikellorza Thank you!

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:stop

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:forward
